packages/sysutils/udevil/package.mk

```diff
@@ -26,6 +26,12 @@ post_makeinstall_target() {
 
   mkdir -p ${INSTALL}/usr/bin
     cp -PR src/udevil ${INSTALL}/usr/bin
+
+  mkdir -p ${INSTALL}/usr/sbin
```

Comment:
why is this needed for ntfs and not for ext4 etc ?

Reply:
The -t doesn’t match (ntfs != ntfs3) and udevil checks /proc/filesystems - https://github.com/IgnorantGuru/udevil/issues/103 it is definitely not elegant. Would be happy for a better idea.